content/en/events/upcoming-events/gsoc-proposal-template.md

```diff
@@ -0,0 +1,66 @@
+# Project Title
```

Comment:
I think we should put this template in the `kubeflow/community` repo, so its not rendered as a page, but actually visible as a markdown file for users to download.

Reply:
I will move this into two PRs, one to the community and one to website

Reply:
https://github.com/kubeflow/community/pull/843